src/script/microbit-interfacing/MicrobitSerial.ts

```diff
@@ -140,15 +136,21 @@ export class MicrobitSerial implements MicrobitConnection {
           accelerometer: true,
           buttons: true,
         });
-        await this.usb.serialWrite(startCmd.message);
```

Comment:
Flagging this as an oddity that's been cleaned up here: previously we send the start command twice, once via the old way that didn't check the results and the new way below.

Reply:
Yes, I should have mentioned via PR comment that I was removing this.
I added it originally with a "TODO" comment indicating that I didn't know why sometime it didn't start streaming unless the START command was sent twice.
I haven't been able to replicate the issue anymore, so I've removed it, but we should keep an eye in case the periodic messages sometimes don't start streaming.